src/app/views/sap/create_mission.php

```diff
@@ -14,13 +14,12 @@
                 <h1>Create new mission</h1>
                 <?php
                 if (isset($errors) && count($errors) !== 0) {
-                    foreach ($errors as $error) {
-                        echo "<p style=\"color:red\">$error</p>";
-                    }
-                } else if (isset($result) && $result === false) {
-                    echo "<p style=\"color:red\">Something went wrong. :/</p>";
-                }
-                ?>
+                    foreach ($errors as $error) { ?>
```

Comment:
> I just saw that.. The php tags should also be indented. In this case you've ignored that.

You mean `<?php` should be indented one level? Or the code inside that should be indented one level?

> And doing all of this might make things messy.
That's exactly why I used the earlier `echo` syntax. It actually made things cleaner.

> I like the colon syntax more. It's more clear

Again, I agree. But even within another block of PHP?